README.md

```diff
@@ -93,6 +93,9 @@
 
 | Presentation Title | Author(s) | Year |
 | --- | --- | --- |
+| [Test your tests: the do's and don'ts of testing](https://youtu.be/tZDV47JzSw0?si=wK0IMoCA5m4HXe-C) | Kurt Willis | 2023 |
+| [Slither: a static analysis tool for Vyper and Solidity](https://youtu.be/UolRTfsFx3Q?si=JPJROuunWn9lfzyV) | Troy Sargent | 2023 |
+| [Roundme: rounding analysis made simpler](https://youtu.be/mgJXzBNPy0U?si=D8VofSoJVR9U7nqk) | Josselin Feist | 2023 |
```

Comment:
Do we have slides for these? If so they should get checked into the `presentations/` folder like the other talks.

Reply:
@woodruffw we have at least some of them, validating that we can save & upload